LovefieldService.js

```diff
@@ -402,12 +401,21 @@ LovefieldService.prototype.selectFilteredResults = function(filters, pathFilters
   });
 }
 
-LovefieldService.prototype.deleteEntries = function() {
-  var q1 = this.db_.delete().from(this.test_results);
-  var q2 = this.db_.delete().from(this.tests);
-  var q3 = this.db_.delete().from(this.test_runs);
-  var tx = this.db_.createTransaction();
-  return tx.exec([q1, q2, q3]);
+LovefieldService.prototype.deleteEntries = function(run_id) {
+  return this.getDbConnection().then((db) => {
+    var q1 = db.delete().from(this.test_results);
```

Comment:
Maybe we could create variables named `tests`, `test_results` and `test_runs` for consistency with the other functions.
Something like `var tests = this.tests` and so on